src/services/utilServices/RateLimiter.ts

```diff
@@ -9,7 +11,13 @@ import rateLimit from "express-rate-limit"
 // but not on the other, leading to inconsistent behaviour.
 // eslint-disable-next-line import/prefer-default-export
 export const rateLimiter = rateLimit({
-  windowMs: 15 * 60 * 1000, // 15 minutes
+  windowMs:
+    parseInt(
+      process.env.AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS ||
+        DEFAULT_AUTH_TOKEN_EXPIRY_MILLISECONDS.toString(),
+      10
```

Comment:
nit: Hey, rather than defining an int, parsing it to string and then parsing it back into an int, could we just use the int directly?

Reply:
we actually can't cos `parseInt` requires a string (despite being able to return `NaN`). 

another alternative is to do `parseInt(... || "", 10) ??  DEFAULT_AUTH_TOKEN_EXPIRY_MILLISECONDS`, which also seems jank. lmk which one you'd rather have

Reply:
Arh, ok. Nah, am ok with this